src/import/importConfiguration.ts

```diff
@@ -893,6 +896,28 @@ function parseInstanceOptions(
   };
 }
 
+function detectPotentialMistakes(yaml: YAMLConfiguration) {
+  const unrecognizedKeys = Object.keys(yaml).filter(
+    potentialKey => !ALL_YAML_CONFIG_PROPERTIES.includes(potentialKey)
+  );
+  if (unrecognizedKeys.length > 0) {
+    const propertyWord = unrecognizedKeys.length === 1 ? 'property' : 'properties';
+    logger.warn(
+      `Unrecognized ${propertyWord} found in configuration: ${unrecognizedKeys.join(', ')}`
+    );
+  }
+  // additional check on pages, since their nameUrl should always end with .md or .xml
+  if (yaml.pages != null) {
+    const unusualPages = Object.keys(yaml.pages).filter(nameUrl => !/\.(md|xml)$/.test(nameUrl));
+    if (unusualPages.length > 0) {
+      const pageWord = unusualPages.length === 1 ? 'Page' : 'Pages';
+      logger.warn(
+        `${pageWord} not ending in .md or .xml found in configuration: ${unusualPages.join(', ')}`
+      );
+    }
+  }
+}
```

Comment:
One configuration file I had that I tried this on had a pages object like so:

```yaml
pages:
  index.html:
    title: Home
    background.xml: 
      title: Background Info
```
I can't remember exact if this will cause issues down the line (and I haven't run the IG Publisher on this yet), but do you know if html files are allowed? Or if `index.html` is a special case?